core/storage/question/gae_models.py

```diff
@@ -246,6 +246,23 @@ def get_models_by_skill_id(cls, skill_id):
         return QuestionSkillLinkModel.query().filter(
             cls.skill_id == skill_id).fetch()
 
+    @classmethod
+    def get_models_by_question_id(cls, question_id):
+        """Returns a list of QuestionSkillLink domains of a particular
+        question ID.
+
+        Args:
+            question_id: str. ID of the question.
+
+        Returns:
+            list(QuestionSkillLinkModel)|None. The list of question skill link
+            domains that are linked to the question ID. None if the question
```

Comment:
Does this correctly differentiate between "question ID doesn't exist" and "there are no question skill link models corresponding to this question ID"? Looking at the implementation I'm not sure it does, we might need to update the docstring to be clearer here.

Reply:
fixed